packages/orchestrator-ui-components/src/components/WFOTable/utils/tableConfigPersistence.ts

```diff
@@ -1,46 +1,42 @@
 import { TableColumnKeys } from './columns';
 
-export type LocalStorageTableConfig<T> = {
+export type StoredTableConfig<T> = {
     hiddenColumns: TableColumnKeys<T>;
     selectedPageSize: number;
 };
 
 export const isValidLocalStorageTableConfig = <T>(
-    object: LocalStorageTableConfig<T>,
-): object is LocalStorageTableConfig<T> =>
+    object: StoredTableConfig<T>,
+): object is StoredTableConfig<T> =>
+    typeof object === 'object' &&
     'hiddenColumns' in object &&
     object.hiddenColumns !== undefined &&
     'selectedPageSize' in object &&
     object.selectedPageSize !== undefined;
 
 export const getTableConfigFromLocalStorage = <T>(
     key: string,
-): LocalStorageTableConfig<T> | undefined => {
-    try {
+): StoredTableConfig<T> | undefined => {
+    if (typeof window !== 'undefined' && window.localStorage) {
```

Comment:
I think the try/catch is still needed. Accessing window.localstorage might throw an error.
http://crocodillon.com/blog/always-catch-localstorage-security-and-quota-exceeded-errors
(I did not test this myself though)